src/components/PageBlocks/FileConfigurator.vue

```diff
@@ -11,10 +12,24 @@
                           :options="modes">
       </b-form-radio-group>
     </b-form-group>
+    <div
+      v-if="options.mode === 'list' || options.mode === 'grid'"
+    >
+      <small>
+        {{ $t('field.kind.file.view.fileExtensions.listAndGrid') }}
```

Comment:
All of this can be solved with a computed property that returns the right string based on the mode. Duplicating templates is the wrong approach